src/main/menStore.java

```diff
@@ -0,0 +1,136 @@
+package main;
+
+import java.util.Scanner;
+
+public class menStore {
+    public static void main(String[] args) {
+        System.out.print("\n*MEN'S STORE*");
+        while (true) {
+            showMenu();
+            menStore();
+
+        }
+    }
+
+    private static void showMenu() {
+        String menu = """
+                \n-MAIN MENU-
```

Comment:
you could use System.out.println method at line 7,  in order to avoid this \n

Reply:
i will explain why i did that.